docs/src/developers_guide/contributing_documentation.rst

```diff
@@ -1,173 +1,21 @@
 
-.. _contributing.documentation:
-
 Contributing to the Documentation
```

Comment:
I'm aware this page has the same title as its child page, but not sure how to rename one of them to provide differentiation whilst keeping them short and accurate. Thoughts?

Reply:
@wjbenfold How's about signalling that this is the gateway or crossroads for readers to choose a path i.e., full vs easy.

How's about renaming the title to something like `How to Contribute Documentation`